From 7de2f0751b126851a003392f2cc08901e84bfd5e Mon Sep 17 00:00:00 2001
From: Alex Stewart <alex.stewart@ni.com>
Date: Fri, 19 Jan 2024 14:45:14 -0500
Subject: [PATCH] CMakeLists: fixup utf8cpp library link

The utf8cpp cmake library namespace is incorrectly identified as
'utf8cpp', instead of the proper 'utf8cpp:utf8cpp'. As a result, cmake
does not link the utf8.h header and compilation fails.

Correct the library namespace.

Signed-off-by: Alex Stewart <alex.stewart@ni.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5471d6cd..48b5c07e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -452,7 +452,7 @@ target_link_libraries(ni_grpc_device_server
    ${_PROTOBUF_LIBPROTOBUF}
    ${CMAKE_DL_LIBS}
    nlohmann_json::nlohmann_json
-   utf8cpp
+   utf8cpp::utf8cpp
    )
 
 set_target_properties(ni_grpc_device_server PROPERTIES BUILD_WITH_INSTALL_RPATH TRUE)
